tests/test_api_v1.py

```diff
@@ -102,3 +106,66 @@ def test_validate_extraneous_params(client, db):
     error_msg = json.loads(rv.data.decode('utf-8'))['error']
     assert 'invalid keyword argument' in error_msg
     assert 'spam' in error_msg
+
+
+@mock.patch('tempfile.TemporaryDirectory')
+@mock.patch('cachito.web.api_v1.Request')
+@mock.patch('cachito.web.api_v1.chain')
+def test_download_archive(
+    mock_chain, mock_request, mock_temp_dir, client, db, app, tmpdir
+):
+    shared_volume = tmpdir.mkdir('shared')
+    shared_workdir = shared_volume.mkdir('ephemeral')
+
+    app.config.update(CACHITO_SHARED_DIR=str(shared_volume))
```

Comment:
You should use `patch.dict` since the `app` fixture is scoped to the session. We don't want this influencing other tests.

Reply:
Cool! Didn't know about this. Thanks.